### DISCUSSION TITLE
can we pull data as per different environment in taiko

### DISCUSSION THREAD

**Discussion by sunaina2022** [getgauge/taiko#2561] [General]
can we pull data as per different environment in taiko

**Comment by sunaina2022** [comment]
can someone please help,this is urgent

**Comment by NivedhaSenthil** [comment]
Taiko does not handle test data, that has to be handled by the test runner being used like Gauge, mocha or jest. You can use different user profile of browser to have the settings and local storages shared if that can help.

**Comment by sswaroopgupta** [comment]
In other words, when Taiko is used with test runners like Gauge, mocha, jest... we can run against different profiles.

**Comment by sunaina2022** [comment]
Thanks Nivedita. Gauge provide flexibility to pick test data from diff path based on env using "gauge_data_dir" key but if i try same thing with gauge+taiko= i do not see options to run specs. This mean specs are not recognizing path used in property file

using above feature Gauge use test data file path from default.properties file and rest of the path is concated from specs. Same thing does not seem to be working in Gauge+Taiko

**Comment by gulshankamboj** [comment]
If you want to run your gauge/taiko test cases against different environments you can use following command:
 gauge run --env {env name here} --tags {tags here} --verbose specs 

You will have to execute test cases of multiple environments one by one.

If you want to create separate env data you can refer to documentation here:
https://docs.gauge.org/configuration.html?os=macos&language=javascript&ide=vscode 

Just type in the search box - Using environments in a Gauge project or Running a Specification 

Your issue is not very descriptive. Please explain it properly if not resolved.
